### PR TITLE
Replace all parameter references from double backticks to single

### PR DESCRIPTION
**Pull request by klecki**
## Description:
Semi-automatically search for all parameters wrapped in double backticks and replace them with single backticks so that the sphinx automation can recognize them and inject the `:paramref:` directive allowing for proper linking to parameters via their references. 

### Affected modules and functionalities:
Docstring, some deprecation mechanism.

There might be too much changes applied (some strings were used in the errors/warnings), so review of the changes and running the tests is necessary to revert some of the changes.

I see few glitches, will fix them tomorrow.

### Key points relevant for the review:
Please click on the links and check if they work :)

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [x] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: dali/operators/bbox/bb_flip.cc [hunk 1]
```diff
@@ -25,12 +25,12 @@ DALI_SCHEMA(BbFlip)
     .DocStr(R"code(Flips bounding boxes horizontally or vertically (mirror).
 
 The bounding box coordinates for the  input are in the [x, y, width, height] - ``xywh`` or
-[left, top, right, bottom] - ``ltrb`` format. All coordinates are in the image coordinate
+[left, top, right, bottom] - `ltrb` format. All coordinates are in the image coordinate
```
Comment (klecki) [inline dali/operators/bbox/bb_flip.cc:28]: Todo: `ltrb` is a false positive, change to "ltrb"

**Comment by klecki** [comment]
!build

**Comment by klecki** [comment]
!build

**Review by mdabek-nvidia — APPROVED** [review]

**Comment by klecki** [comment]
!build

**Pull request merged**